apps/src/aichat/views/presentation/ModelCardRow.tsx

```diff
@@ -27,8 +27,8 @@ const ModelCardRow: React.FunctionComponent<ModelCardRowProps> = ({
       }
       return (
         <ul>
-          {checkedExpandedContent.map(content => (
-            <li>{content}</li>
+          {checkedExpandedContent.map((content, index) => (
+            <li key={index}>{content}</li>
```

Comment:
Unrelated update to add key to list items here and below.

Reply:
Style update moved to separate PR https://github.com/code-dot-org/code-dot-org/pull/57996